db/compaction_job.cc

```diff
@@ -811,6 +811,7 @@ void CompactionJob::ProcessKeyValueCompaction(SubcompactionState* sub_compact) {
   if (measure_io_stats_) {
     prev_perf_level = GetPerfLevel();
     SetPerfLevel(PerfLevel::kEnableTime);
+    perf_context.EnablePerLevelPerfContext();
```

Comment:
I don't understand why we need it. Is there per-level perf context counters we report for compactions.